e2e-tests/csv-source/upload.hydra

```diff
@@ -59,17 +59,29 @@ With Class cc:CubeProject {
 
                 Invoke {
                     Content-Type "text/csv"
-                    Content-Disposition "file; filename='test.jpg'"
+                    Content-Disposition "file; filename='test2.jpg'"
 
                     ```
-                    A,B,C
+                    C,D,E
                     1,2,test
                     ```
                 } => {
                     Expect Status 201
                     Expect Type <https://cube-creator.zazuko.com/vocab#CSVSource>
                     Expect Property <http://schema.org/associatedMedia>
                 }
+
+                Invoke {
+                    Content-Type "text/csv"
+                    Content-Disposition "file; filename='test.jpg'"
+
+                    ```
+                    A,B,C
+                    1,2,test
+                    ```
+                } => {
+                    Expect Status 409
```

Comment:
I never used 409, is this a usual way of telling the user that a file already exists?

Reply:
@tpluscode suggested it and I liked it. If you have an other preference, tell me :wink: I am not attached to it.

Reply:
By the way I have not touched the front end for the error handling (yet?)

Reply:
I pushed a change for the UI

Reply:
> [The 409 (Conflict) status code indicates that the request could not be completed due to a conflict with the current state of the target resource. This code is used in situations where the user might be able to resolve the conflict and resubmit the request. The server SHOULD generate a payload that includes enough information for a user to recognize the source of the conflict.](http://tools.ietf.org/html/rfc7231#section-6.5.8)

We could address the `SHOULD` clause by linking to the already existing resource. In such case the client could direct the user to make a `PUT` instead to update the source. 

I would have the as a separate PR though. Not only we do not have the update operation yet, it would also be nice to make `problem+json` payloads proper RDF